File: src/Entity/Reclamation.php
<?php

namespace App\Entity;

use App\Repository\ReclamationRepository;
use Doctrine\DBAL\Types\Types;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

#[ORM\Entity(repositoryClass: ReclamationRepository::class)]
class Reclamation
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private ?int $id = null;

     

    #[ORM\Column(length: 255)]
    #[Assert\NotBlank(message:"le champ est vide")]
    private ?string $objet = null;

    #[ORM\Column(length: 255)]
    #[Assert\NotBlank(message:"le champ est vide")]
    private ?string $description = null;

    #[ORM\Column(length: 255)]
    #[Assert\NotBlank(message:"le champ est vide")]
    private ?string $nom = null;

    public function getId(): ?int
    {
        return $this->id;
    }
 

    public function getObjet(): ?string
    {
        return $this->objet;
    }

    public function setObjet(string $objet): self
    {
        $this->objet = $objet;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(string $description): self
    {
        $this->description = $description;

        return $this;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }


    public function setNom(string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }


    public function __toString()
    {   return $this->id  ; 
        return $this->objet  ; 
        return $this->description;
        return $this->nom;
    }


  

}

File: final/mobile/CodeNameOne-Formation-Youtube-main/src/com/mycomany/entities/Reclamation.java
package com.mycomany.entities;

import java.util.Date;
import com.mycomany.utils.*;

public class Reclamation {
     private int id;
       private String objet,description;

    /*public Reclamation(int id, String email, String password) {
        this.id = id;
        
        this.email = email;
        this.password = password;
        
    }*/

    public Reclamation(){}
    public Reclamation(int id, String objet, String description) {
        this.id=id;
        this.objet = objet;
        this.description = description;
    }
     public Reclamation(String objet, String description) {
        this.objet = objet;
        this.description = description;
    }

    
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getObjet(){
        return objet;
    }

    public void setObjet(String objet) {
        this.objet = objet;
    }
    
    public String getDescription() {
        return description;
    }
       public void setDescription(String description) {
        this.description = description;
    }
  
    @Override
    public String toString() {
        return "reclamation{" + "id=" + id + " objet=" + objet + ", description=" + description +  '}';
    }

    
    
    
    




    

   
    
    
    
}

    
    
    




    

   
    
    
    
